metadata-ws/src/test/resources/features/publication.feature

```diff
@@ -0,0 +1,30 @@
+Feature: publication object
+
+  Scenario: register a publication successfully
+    When I request POST /publications with JSON payload:
+    """
+    {
+      "publicationId": "Publication1"
+    }
+    """
+    Then set the URL to PUBLICATION
```

Comment:
This is still part of what you do, not the expected outcome, so it should be `and` instead of `then`.

Reply:
Corresponding to this fix, i have done minor changes